Add support for clang json format (#1188)
gcovr/gcovr: CHANGELOG.rst

New features and notable changes:

- Due to added support for LLVM source based code coverage format some configuration file

>   keys got changed. The options still exist with ``gcov`` in it but they also affect the

Reviewer: ```suggestion
  keys got changed. The old ``gcov`` named options are still supported but they also affect the
```
